src/Control/Email/Email.php

```diff
@@ -31,738 +16,646 @@ class Email extends ViewableData
 {
 
     /**
-     * @var string $from Email-Address
+     * @var array
+     * @config
      */
-    protected $from;
+    private static $send_all_emails_to = array();
 
     /**
-     * @var string $to Email-Address. Use comma-separation to pass multiple email-addresses.
+     * @var array
+     * @config
      */
-    protected $to;
+    private static $cc_all_emails_to = array();
 
     /**
-     * @var string $subject Subject of the email
+     * @var array
+     * @config
      */
-    protected $subject;
+    private static $bcc_all_emails_to = array();
 
     /**
-     * Passed straight into {@link $ss_template} as $Body variable.
-     *
-     * @var string $body HTML content of the email.
+     * @var array
+     * @config
      */
-    protected $body;
+    private static $send_all_emails_from = array();
 
     /**
-     * If not set, defaults to converting the HTML-body with {@link Convert::xml2raw()}.
+     * This will be set in the config on a site-by-site basis
      *
-     * @var string $plaintext_body Optional string for plaintext emails.
-     */
-    protected $plaintext_body;
-
-    /**
-     * @var string $cc
-     */
-    protected $cc;
-
-    /**
-     * @var string $bcc
-     */
-    protected $bcc;
-
-    /**
-     * @var array $customHeaders A map of header-name -> header-value
-     */
-    protected $customHeaders = array();
-
-    /**
-     * @var array $attachments Internal, use {@link attachFileFromString()} or {@link attachFile()}
+     * @config
+     * @var string The default administrator email address.
      */
-    protected $attachments = array();
+    private static $admin_email = null;
 
     /**
-     * @var boolean $parseVariables_done
+     * @var Swift_Message
      */
-    protected $parseVariables_done = false;
+    private $swiftMessage;
 
     /**
-     * @var string $ss_template The name of the used template (without *.ss extension)
+     * @var string The name of the used template (without *.ss extension)
      */
-    protected $ss_template = 'GenericEmail';
+    private $template = "SilverStripe\\Email\\Email";
```

Comment:
Can you please use the full qualified path for the template? The convention is that it should match the same class name.

I.e.

`private static $template = 'SilverStripe\Control\Email\Email`, or just `private static $template = self::class`.